src/names/common.h

```diff
@@ -446,4 +448,33 @@ class CNameCache
 
 };
 
+/* ************************************************************************** */
+/* CQueuedTransaction.  */
+
+/**
+ * Keeps track of raw transactions that are queued for later broadcast.
+ * Example use cases:
+ *     Broadcast a name_firstupdate when its input name_new reaches maturity.
+ *     Broadcast a name_update when its input name_anyupdate nears expiration.
+ *     Broadcast a name_firstupdate when someone else's name expires.
+ * This data is serialized to the wallet so that the transaction can be
+ * broadcasted even if the client has restarted since the transaction was
+ * queued.
+ */
+class CQueuedTransaction
```

Comment:
Do we even need this class?  It seems that all the transaction queue does is just store raw transactions in the wallet - and in fact it could be used independently of Namecoin, right?  So maybe we should just remove this class and use `CTransaction` directly in the wallet code?  Or is there a reason why you think we may need to extend the class in the future?

Reply:
Hmm, no, this just seems like path-dependency.

Reply:
I'll be using CMutableTransaction, though.

Reply:
Ok cool.  Just from reviewing the code it didn't seem clear to me why it has to be `CMutableTransaction`, but if that's the case for some reason, there's nothing bad with that either I guess.

Reply:
If I recall correctly, it had to do with serialization.